src/main/java/world/bentobox/bentobox/api/addons/AddonClassLoader.java

```diff
@@ -107,7 +107,7 @@ public static AddonDescription asDescription(YamlConfiguration data) throws Inva
             throw new InvalidAddonDescriptionException("Missing 'authors' tag. At least one author must be listed in addon.yml");
         }
 
-        AddonDescription.Builder builder = new AddonDescription.Builder(
+        AddonDescriptionBuilder builder = new AddonDescriptionBuilder(
```

Comment:
Moving the Builder out of the AddonDescription class is in theory okay, but `AddonDescription.Builder` is called by a lot of test classes in other projects. Those could be adjusted to use this, but be careful when making changes to public APIs.